client/app/index.js

```diff
@@ -71,7 +71,7 @@ import 'script-loader!@bower_components/international-phone-number/releases/inte
 import 'script-loader!@bower_components/qrcode/lib/qrcode.js';
 import 'angular-qr';
 import 'script-loader!u2f-api-polyfill/u2f-api-polyfill.js';
-import 'ovh-angular-otrs';
+import '@ovh-ux/ng-ovh-otrs';
```

Comment:
Could you move this import statement directly to `client/app/app.js`?

```suggestion
import ngOvhOtrs from '@ovh-ux/ng-ovh-otrs';
```

Reply:
This is done :)

Reply:
Thank you!